qa/tasks/kafka.py

```diff
@@ -109,20 +110,29 @@ def run_kafka(ctx,config):
         for (client, _) in config.items():
             (remote,) = ctx.cluster.only(client).remotes.keys()
 
+            ctx.cluster.only(client).run(args=['pgrep', '-a', 'java', run.Raw('|'), 'grep', 'zookeeper.properties', run.Raw('|'), 'awk', '{print $1}'])
```

Comment:
i think there might be a more efficient command to do that. look here: https://man7.org/linux/man-pages/man1/pgrep.1.html

something like: `pkill -f zookeeper.properties || :`
this would just kill any command that has "zookeeper.properties" in its commandline
use the ` || :` part to make sure that you always return zero - even if `pkill` does not find a matching process

Reply:
but please call that after you call the "stop" script

Reply:
yeah thanks, I'll check it out